tests/mocha/field_variable_test.js

```diff
@@ -251,7 +251,7 @@ suite('Variable Fields', function() {
         const field = new Blockly.FieldVariable(
             'test', undefined, ['Type1'], 'Type1');
         chai.assert.deepEqual(field.variableTypes, ['Type1']);
-        chai.assert.equal(field.defaultType_, 'Type1');
```

Comment:
I could see it, but not if it's only for testing purposes. Have we otherwise been getting requests for it for these properties?

Reply:
Yeah not that I know of, it just disgruntles me to be accessing private things in tests :P But the ideal solution is probably testing the actual public-facing effects of these constructor values (e.g. what things show up in the drop down) rather than doing what we're currently doing, or creating an accessor. Which, eh, probably too much work haha.